lib/fluent/plugin/out_sumologic.rb

```diff
@@ -86,6 +86,9 @@ class Fluent::Plugin::Sumologic < Fluent::Plugin::Output
   config_param :timestamp_key, :string, :default => 'timestamp'
   config_param :proxy_uri, :string, :default => nil
   config_param :disable_cookies, :bool, :default => false
+  # https://help.sumologic.com/Manage/Fields
+  desc 'Fields string (eg "a=b,c=d") which is going to be added to every record.'
```

Comment:
Could have used a better example
e.g `cluster=payment, service=credit_card`

Reply:
Sure, will do that